test/integration/test-shadow-amp.js

```diff
@@ -22,7 +22,6 @@ describes.integration(
     body: `
       <!-- unminified src for local-tests.js -->
       <script async src="/dist/amp-shadow.js"></script>
-      <!-- minified src for single-pass-tests.js -->
```

Comment:
Pretty sure the following line must be deleted too (looks like you only deleted the comment).